### PR TITLE
TEST FIX, CLEAN UP

### PR DESCRIPTION
**Pull request by DSsoto**
* Remove console output from alarm server during rostest
* Clean up camakelists
* Fix bug with cpp .test file

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -44,28 +43,22 @@ catkin_package(
...
+    message_runtime
 )
 
 include_directories(
   ${catkin_INCLUDE_DIRS}
 )
+
 # Declare a C++ library
-set(SRCS
+set(ros_alarms_srcs
```
Comment (ironmig) [inline CMakeLists.txt]: convention seems to be ALL_CAPS for variables like these to differentiate from targets

File: CMakeLists.txt [hunk 2]
```diff
@@ -44,28 +43,22 @@ catkin_package(
...
-
-add_dependencies(ros_alarms ros_alarms_generate_messages_cpp)
-
-#if(CATKIN_ENABLE_TESTING)
-#  foreach(T
-#    test/test_ros_alarms.test)
-#     add_rostest(${T})
-#  endforeach()
-#endif()
+add_dependencies(ros_alarms ${PROJECT_NAME}_generate_messages_cpp ${catkin_EXPORTED_TARGETS})
```
Comment (ironmig) [inline CMakeLists.txt]: Should be ```${${PROJECT_NAME}_EXPORTED_TARGETS}``` as per [ros wiki](http://docs.ros.org/hydro/api/catkin/html/howto/format2/building_msgs.html)

File: test/ros_alarms_cpp.test [hunk 3]
```diff
@@ -10,5 +10,5 @@
       test_alarm
   </rosparam>
 
-  <test test-name="ros_alarms_cpp_tests" pkg="ros_alarms" type="ros_alarms_integration_tests" />
+  <test test-name="ros_alarms_integration_tests" pkg="ros_alarms" type="ros_alarms_cpp_tests" />
```
Comment (ironmig) [inline test/ros_alarms_cpp.test]: To this day they still don't know how this worked on CI

File: CMakeLists.txt (hunk 2 above)
Comment (DSsoto) [inline CMakeLists.txt]: Both work but yeah, that is the right way.

File: CMakeLists.txt (hunk 1 above)
Comment (DSsoto) [inline CMakeLists.txt]: Done

**Pull request merged**